source/start/topics/recipes/wordpress.rst

```diff
@@ -202,7 +202,8 @@ Rewrite rules for Multisite using subdomains
 
 .. note::
 
-    * For wordpress-nginx based sites management, `EasyEngine <https://github.com/rtCamp/easyengine>`_ can be used. EasyEngine is collection of shell scripts for Ubuntu.
+    * For wordpress-nginx based sites management, `EasyEngine <https://github.com/rtCamp/easyengine>`_ can be used. EasyEngine (ee) is python based command line control panel to setup NGINX server on Ubuntu and Debian Linux distribution
+for HTML, PHP, MySQL, HHVM, PageSpeed and WordPress sites.
```

Comment:
the 'for' from this line needs to be indented to the first character of the 'For' from the line above. This is so that the system knows that it is part of the same bullet point. Like you would with Python code.